### PR TITLE
User can open finalized workout in printable format.

### PR DESCRIPTION
**Pull request by johnhaup**
screen shot 2017-06-21 at 2 14 16 pm
[testprint.pdf](https://github.com/johnhaup/crunchtime/files/1092968/testprint.pdf)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -49,8 +49,8 @@ <h2 class="text-center">Total Workout Time</h2>
         </div>
       </div>
       <div class="row">
-        <div class="col-sm-7 main-divs">
-          <div class="panel panel-default ex-panel">
+        <div class="col-sm-7" id="chooseEx">
```
Comment (victormejia) [inline index.html]: `choose-ex` here for id

File: index.html [hunk 2]
```diff
@@ -49,8 +49,8 @@ <h2 class="text-center">Total Workout Time</h2>
...
+          <div class="panel panel-default bottom-divs">
```
Comment (victormejia) [inline index.html]: is there something more semantic you can name it vs. `bottom-divs`?

File: main.js [hunk 3]
```diff
@@ -131,7 +131,9 @@ function addToWorkout(newEx) {
   var $info = document.createElement('div')
   $info.className = 'panel-body'
   createPanelBody($info, newEx)
-  createPanelFoot($info, newEx)
+  if (newEx.name !== newEx.alt && newEx.muscle !== 'Abs') {
```
Comment (victormejia) [inline main.js]: expand on `alt`, not sure what that is

**Comment by victormejia** [comment]
also the gif isn't displaying correctly

**Comment by johnhaup** [comment]
I think I already removed it a few commits ago.  I deleted it from terminal and added changed file to my commit.  It shows red blocked and "Binary File Removed" tooltip when I look at the files for this pull request.

**Comment by victormejia** [comment]
you will have to use `git rm`

**Pull request merged**